# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../luaexpat/no-config.patch
# Copyright (C) 2006 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- luaexpat-1.0.1-vanilla/config	2005-06-05 02:29:27.000000000 +0200
+++ luaexpat-1.0.1/config	1970-01-01 01:00:00.000000000 +0100
@@ -1,34 +0,0 @@
-# Installation directories
-# System's libraries directory (where binary libraries are installed)
-LUA_LIBDIR= /usr/local/lib/lua/5.0
-# System's lua directory (where Lua libraries are installed)
-LUA_DIR= /usr/local/share/lua/5.0
-# Lua includes directory
-LUA_INC= /usr/local/include/lua5
-# Expat includes directory
-EXPAT_INC= /usr/local/include
-
-# OS dependent
-LIB_OPTION= -shared #for Linux
-#LIB_OPTION= -bundle -undefined dynamic_lookup #for MacOS X
-
-LIBNAME= lib$T-$V.so
-COMPAT_DIR= ../compat/src
-
-# Compilation parameters
-# On FreeBSD systems, the following line should be commented
-DLLIB= -ldl
-#LUA_LIBS= -llua50 -lm
-CWARNS = -Wall -pedantic \
-        -Waggregate-return \
-        -Wcast-align \
-        -Wmissing-prototypes \
-        -Wstrict-prototypes \
-        -Wnested-externs \
-        -Wpointer-arith \
-        -Wshadow \
-        -Wwrite-strings
-
-CFLAGS = $(CONFIG) $(CWARNS) -ansi -O2 -I$(LUA_INC) \
-   -I$(COMPAT_DIR) -I$(EXPAT_INC)
-CC = gcc
